README.md

```diff
@@ -1 +1,37 @@
-LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
\ No newline at end of file
+# learnaway
+
+LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
+
+## learnaway-api
+
+### Requirements
+
+Requires docker-compose 
```

Comment:
For now single container is enough for the project, so we don't really need `docker-compose`. But I see the benefit in using it already: simpler setup and easier extension later on; so ok with it :+1: